src/classes/json_data.py

```diff
@@ -263,6 +263,11 @@ def replace_string_to_relative(self, match):
         key = match.groups(0)[0]
         path = match.groups(0)[1]
         folder_path, file_path = os.path.split(os.path.abspath(path))
+        project_win_drive = os.path.splitdrive(get_app().project.current_filepath)[0]
+        file_win_drive = os.path.splitdrive(path)[0]
+        if file_win_drive != project_win_drive:
+            # IF the file is on different drive. Don't abbreviate the path.
+            return path
```

Comment:
The logic here seems ok, but it needs to come **_after_** the `else` on line 308. All of the changes before that point still need to happen. 

(Things like replacing the transitions path with `\@transitions`, which is always valid no matter what drive everything is on.)

Reply:
In theory, this would only be true for files on different drive letters.. and thus, it shouldn't matter before or after the `else` on 308, right?  Regardless, it might be easier logic to follow with this logic after the `else`.

Also, what happens on Linux and Mac with this function call? `os.path.splitdrive`

Reply:
`os.splitdrive('/home/user/Videos') ` :arrow_right: `[ '', '/home/user/Videos`]`

Reply:
And I'll get testing on how this treats transition paths

Reply:
@jonoomph The code was doing an early `return` if the drive letters differed, which would've prevented the stuff before that `else` from executing. But, yeah, on Mac/Linux it's fine, the "drive letter" will always be `""` for all paths.

Reply:
(If the project is being saved on a drive that's _not_ the `C:\` drive, the drive letter **will** differ between the transition path (`C:\Program Files\OpenShot Video Editor\transitions\foo.png`) and the project file path (`F:\My Projects\something.osp`), but we still want `C:\Program Files\OpenShot Video Editor\transitions\` to be replaced with `@transitions`.)

Reply:
I moved it to inside the else. So it should still handle transitions normally (if it wasn't)